include/epanet.cs

```diff
@@ -0,0 +1,662 @@
+//using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+//epanet2.cs[By Oscar Vegas]
+//Last updated on 17/09/2020
+
+//Declarations of functions in the EPANET PROGRAMMERs TOOLKIT
+//(EPANET2.DLL) for use with Visual Basic (C#)
```

Comment:
Please change "for use with Visual Basic (C#)" to "for use with C#" since we already have a header for VB.

Reply:
Comment line corrected